deploy/contents/k8s/cp-release-notes/cp-release-notes-dpl.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: cp-release-notes
+  namespace: default
+spec:
+  schedule: ${CP_RELEASE_NOTES_SCHEDULE}
+  startingDeadlineSeconds: 300
+  successfulJobsHistoryLimit: 5
+  failedJobsHistoryLimit: 5
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          namespace: default
+          labels:
+            cloud-pipeline/cp-release-notes: "true"
+        spec:
+          nodeSelector:
+            cloud-pipeline/cp-release-notes: "true"
+          tolerations:
+            - key: node-role.kubernetes.io/master
+              effect: NoSchedule
+          containers:
+            - name: cp-release-notes
+              image: ${CP_DOCKER_DIST_SRV}lifescience/cloud-pipeline:release-notes-$CP_VERSION
+              securityContext:
+                privileged: true
+              imagePullPolicy: "Always"
+              command: ["/init"]
+              envFrom:
+                - configMapRef:
+                    name: cp-config-global
+              volumeMounts:
+                - mountPath: /opt/release-notes/logs
+                  name: release-notes-logs
+                - mountPath: /opt/release-notes/templates
+                  name: release-notes-template
+                - mountPath: /opt/release-notes/data
+                  name: release-notes-data
+          volumes:
+            - name: release-notes-logs
+              hostPath:
+                path: /opt/release-notes/logs
+            - name: release-notes-template
+              hostPath:
+                path: /opt/release-notes/templates
+            - name: release-notes-data
+              hostPath:
+                path: /opt/release-notes/data
+          restartPolicy: OnFailure
```

Comment:
I believe that actually we don't need to restart again and again the job if it is fails, because it isn't a critical service and with `restartPolicy: OnFailure` we have a risk to stack in a infinity loop if something goes wrong

Reply:
lets explicitly disable it